tests/test_browser.py

```diff
@@ -2582,3 +2582,93 @@ def test_modularize(self):
         ''' % code)
         self.run_browser('a.html', '...', '/report_result?0')
 
+  # Test that the emscripten_ atomics api functions work.
```

Comment:
No, currently these tests will throw an exception and fail. Probably will have to do something about this. It's slightly annoying that the early-out skip exit will have to live e.g. in src/preamble.js since otherwise the code will throw there when it tries to access nonexisting SharedArrayViews.